test/ui/cypress/CYPRESS.md

```diff
@@ -0,0 +1,72 @@
+# Cypress Testing
```

Comment:
This content is misleading, it suggests to setup python environment but that is only needed when running cypress against locally running deployment driver.
Also, this content should be merged into main readme.

Reply:
> This content is misleading, it suggests to setup python environment but that is only needed when running cypress against locally running deployment driver. Also, this content should be merged into main readme.

@tznamena , I made [the change](https://github.com/ansible/azure-aap-deployment-driver/pull/116/files#diff-8af617e1cdf4c7bb711bd4c31f9f90262c780272349fb4341dc90dc36e0da7f3R60) in Readme to mention the guide of `CYPRESS.md`. 
Having a python virtual env to install dependencies then run tests is a good practice. If this content is misleading, I can make some change for it to clearly mention it is optional.

Reply:
We can use python in E2E as well. For QE's testing, we will refer to CYPRESS.md to set up environment. Having a separate document for it and linking it with the main README file looks better to me.

Reply:
Having two different files make maintenance more complexity. We already have one readme file setup, let's use that one to keep instructions in one place.

Reply:
@tznamena , my concern is merging `CYPRESS.md` into `Readme.md` as one place will bring some confusion to QE. The `Readme.md` will be mainly used by the developers. And the `CYPRESS.md` will be mainly used by QE. I made some changes in the `Readme.md` to add a documentation link to the `CYPRESS.md`. If the developers want to run automation without mocker, they can click the documentation link to refer to `CYPRESS.md`. 
In the platform UI repo, they also use a separate documentation for the automation in the `cypress` folder.

Reply:
Let's keep all the documentation in one place. There should not be any separation of documentation for different types of users, teams, etc. This is an open source project!

Reply:
Seriously, how long are we going to debate this? Let's put all the instructions in one place, please.

Reply:
Splitting the documentation is going to be source of confusion and we will have more files to maintain when things change. 
Please put it all in one file and when people come to us saying that they are confused by having documentation in one place we can split it.

Also, the section "1. Initialize your local environment." shouldn't be there at all because it is no relevant to running cypress at all.

Reply:
@tznamena , if this is an open source project, how about that you create a CONTRIBUTING.md to outline items the contributors should comply and follow. I will refer to it to refine my PR.

Reply:
Excellent point, I didn't notice it missing, I will add it. In the meantime, please keep all the instructions in the main readme file.